### PR TITLE
Fix Tpyos

### PR DESCRIPTION
**Pull request by SyntaxNode**

### CODE REVIEW

File: adapters/infoawarebidder_test.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ func TestImpFiltering(t *testing.T) {
 		}
 
 		// Extra MakeRequests() call check: our mockBidder returns an adapter request for every imp
-		assert.Len(t, actualAdapterRequests, test.expectedImpLen, "Test failed. Incorrect lenght of filtered imps: %s", test.description)
+		assert.Len(t, actualAdapterRequests, test.expectedImpLen, "Test failed. Incorrect length of filtered imps: %s", test.description)
```
Comment (guscarreon) [inline adapters/infoawarebidder_test.go:176]: I spell `length` wrong all the time. 😄

**Review by guscarreon — APPROVED** [review]
Hilarious PR name 😄

File: adapters/infoawarebidder_test.go (hunk 1 above)
Comment (SyntaxNode) [reply adapters/infoawarebidder_test.go:176]: I apparently spell parameters wrong all the time. :) I noticed most of these typos are from me. Doh!

**Review by bsardo — APPROVED** [review]
LGTM

**Pull request merged**